### DISCUSSION TITLE
文件上传错误

### DISCUSSION THREAD

**Discussion by Youngclub** [AlistGo/alist#4763] [Q&A]
上传文件错误无提示： Cannot read properties of undefined (reading 'status')

**Comment by Mooling0602** [comment]
+1

**Comment by pongfcnkl** [comment]
添加本地存储，用网页上的复制功能上传

**Reply by Mooling0602** [reply]
我用本地存储也报这个怎么说

**Reply by pongfcnkl** [reply]
> 我用本地存储也报这个怎么说

什么网盘？

**Reply by Mooling0602** [reply]
> > 我用本地存储也报这个怎么说
> 
> 什么网盘？

就AList里面的本地存储驱动啊，传了几十MB就会出现这个报错

**Reply by pongfcnkl** [reply]
> > > 我用本地存储也报这个怎么说
> > 
> > 
> > 什么网盘？
> 
> 就AList里面的本地存储驱动啊，传了几十MB就会出现这个报错

只传到硬盘，不传网盘？
你是不是反代过了，比如cf nginx

**Reply by Mooling0602** [reply]
> > > > 我用本地存储也报这个怎么说
> > > 
> > > 
> > > 什么网盘？
> > 
> > 
> > 就AList里面的本地存储驱动啊，传了几十MB就会出现这个报错
> 
> 只传到硬盘，不传网盘？ 你是不是反代过了，比如cf nginx

nginx

**Reply by pongfcnkl** [reply]
> > > > > 我用本地存储也报这个怎么说
> > > > 
> > > > 
> > > > 什么网盘？
> > > 
> > > 
> > > 就AList里面的本地存储驱动啊，传了几十MB就会出现这个报错
> > 
> > 
> > 只传到硬盘，不传网盘？ 你是不是反代过了，比如cf nginx
> 
> nginx

把nginx去除了，直连alist吧

**Comment by RSJWY** [comment]
这个问题我这边虽然提示报错，但是上传是成功了的
版本：v3.32.0
存储：S3 上传是挂载了另一个站点的S3，也就是 V3——>S3
